Assets/Scripts/Gameplay/UserInput/ClientInputSender.cs

```diff
@@ -23,7 +23,9 @@ public class ClientInputSender : NetworkBehaviour
         //upstream network conservation. This matters when holding down your mouse button to move.
         const float k_MoveSendRateSeconds = 0.04f; //25 fps.
 
-        const float k_TargetMoveTimeout = 0.45f;  //prevent moves for this long after targeting someone (helps prevent walking to the guy you clicked).
+        const float
+            k_TargetMoveTimeout =
+                0.45f; //prevent moves for this long after targeting someone (helps prevent walking to the guy you clicked).
```

Comment:
@Elfi0Kuhndorf  I see a lot of weird formatting changes in this file: this makes a lot of noise in the PR, and raises the chances to create conflicts.
Can you please revert all the formatting changes, and keep only what's relevant to the fix?

P.S: are you usign some code formatter? It's weird that it breaks declarations over 3 lines, this makes the code unreadeable in the long run.

Reply:
Oh I guess my rider does not use proper standards then :( I was using the "Reformat" option in Rider, I guess I will revert that change then and just manually delete the line which was an issue. Is there a way you all in the team format the code? (so that it is consistent)

Reply:
I'm old-school and don't use rider, maybe @fernando-cortez could provide help on this?